packages/katello/katello/backup.rb

```diff
@@ -324,7 +324,14 @@ def generate_metadata
     end
 
     def create_pulp_data_tar
-      run_cmd("tar --selinux --create --file=#{File.join(@dir, 'pulp_data.tar')} --exclude=var/lib/pulp/katello-export --listed-incremental=#{File.join(@dir, '.pulp.snar')} --transform 's,^,var/lib/pulp/,S' -S *")
+      @archive_size = @options[:split_pulp_tar] || "0"
```

Comment:
The `|| 0` isn't needed, `@options[:split_pulp_tar]` will resolve to nil if nothing was passed, and it can be used in conditionals

Reply:
Neat!  I'll switch to that.